src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -38,6 +38,8 @@ datasets:
   ferc714:
     years: [2019, 2020]
   eia:
+    eia176:
+      years: [2020, 2022]
```

Comment:
The `description` associated with this doesn't match up at all -- I can make it more generic if that would be helpful.

Reply:
Which `description` are you referring to?

Reply:
In this file (`etl_fast.yml`), the key-value pair:

```
description: >
  FERC 1 data from 2020 and 2021, EIA 860/923 from 2020 and 2022 (output to SQLite) plus
  EPA CEMS hourly emissions data from 2020 and 2022 (output to Parquet).
```

Reply:
Oh yes, good catch! That definitely needs to get updated. How about:

> A smaller subset of data to test the ETL. This includes the most recent years of XBRL and DBF data for FERC data, and corresponding years for EIA and EPA CEMS data. FERC and EIA data is output to SQLite, and EPA CEMS hourly emissions data is output to Parquet.

Reply:
Missed your rewrite of the description, which looks just fine as well!